app/controllers/tags_controller.rb

```diff
@@ -43,14 +44,18 @@ def destroy
 
   private
 
+  def sortable_class
+    Tag
+  end
+
   def tag_params
     modified_params = params.require(:tag).permit(:name, :color)
     modified_params[:name] = "#{modified_params[:color].gsub('#', '!')}_#{modified_params[:name]}"
     modified_params.except(:color)
   end
 
   def set_columns
-    default_field_names = ['Name'].freeze
+    default_field_names = ['Sort', 'Name'].freeze
```

Comment:
Are we using the `Sort` column? I'm not seeing it in the options

Reply:
The `th` is hidden in the view but we need this column to show the sort handle